pkg/importer/dataStream.go

```diff
@@ -37,59 +38,59 @@ type DataStreamInterface interface {
 	Close() error
 }
 
-var _ DataStreamInterface = &dataStream{}
+var _ DataStreamInterface = &DataStream{}
 
-// implements the ReadCloser interface
-type dataStream struct {
-	Url         *url.URL
-	Readers     []Reader
+// DataStream implements the ReadCloser interface
+type DataStream struct {
+	url         *url.URL
```

Comment:
Well... DataStream was not exported, that's something I added/changed because we return it in multiple exported functions, but we were returning a pointer to a non-exported item (dataStream).  I just modifed that, and the name was again a "non" linting name (Url), could be URL or url I went with url because there was no place I could find in the code that accessed it.

Reply:
I appreciate your analysis on which types, funcs, fields, etc should be exported vs. private! Jon and I put some thought in that but also some of the export vs private stuff just evolved as testing was added.